src/external/token/NativeIssuance_v1.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import {Ownable} from "@oz/access/Ownable.sol";
+import {Address} from "@oz/utils/Address.sol";
+import {IERC20Metadata} from "@oz/token/ERC20/extensions/IERC20Metadata.sol";
+
+import {INativeMinter} from "@ex/token/INativeMinter.sol";
+import {INativeIssuance_v1} from "@ex/token/INativeIssuance_v1.sol";
+
+/**
+ * @title    Native Token Issuance
+ * @notice   This contract facilitates the issuance of native tokens on Avalanche L1s. It is designed to be used only by whitelisted minters, ensuring controlled token creation and destruction.
+ * @dev      The contract adheres to the ERC20 standard for interoperability with Inverter Network contracts.
+ *               - Maintains a list of allowed minters for managing who can call mint and burn functions.
+ *               - Allows minting and burning of tokens by members of this whitelist.
+ *               - Mints native tokens through Avalanche Subnet-EVM's Precompiled Native Minter contract.
+ *               - Tokens burned are transferred to a designated burn address, with no recovery option for these tokens.
+ *               - The `depositNative` function is used to deposit native tokens into the contract, which can then be burned by calling the `burn` function to send them to the burn address.
+ * @author Inverter Network
+ */
+contract NativeIssuance_v1 is INativeIssuance_v1, IERC20Metadata, Ownable {
+    using Address for address payable;
+
+    /**
+     * @notice The address of the native minter precompile.
+     */
+    INativeMinter public constant NATIVE_MINTER =
+        INativeMinter(0x0200000000000000000000000000000000000001);
+
+    /**
+     * @notice The address where native tokens are sent in order to be burned for the bonding curve.
+     *
+     * @dev This address was chosen arbitrarily.
+     */
+    address public constant BURNED_FOR_BONDINGCURVE_ADDRESS =
```

Comment:
why not use something like 0xDEAD ?

Reply:
It can be set as 0xDEAD or zero. I wanted to keep it different than those options just to see the amount of burned tokens by the Funding Manager.

I can update it to 0xDead, if it's more convenient.